sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIClient.java

```diff
@@ -715,89 +712,6 @@ public IterableStream<ChatCompletions> getChatCompletionsStream(String deploymen
         return new IterableStream<>(chatCompletionsStream.getEvents());
     }
 
-    /**
-     * Starts the generation of a batch of images from a text caption.
-     *
-     * @param imageGenerationOptions Represents the request data used to generate images.
-     * @throws IllegalArgumentException thrown if parameters fail the validation.
-     * @throws HttpResponseException thrown if the request is rejected by server.
-     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
-     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
-     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
-     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
-     * @return the {@link ImageResponse} for the image generation result.
-     */
-    @ServiceMethod(returns = ReturnType.SINGLE)
-    public ImageResponse getImages(ImageGenerationOptions imageGenerationOptions) {
```

Comment:
We can keep this convenience API for the customer who doesn't want to return with `response`.

Reply:
Naming can be aligned by `getImageGenerations`, or  using `getImagesWithResponse` which drops `Generation` in the response method.